src/marketing/prex-type-visualsim.md

```diff
@@ -0,0 +1,47 @@
+---
+ee_only: true
+title: Visual similarity
+---
+
+The _Visual similarity_ recommendation type recommends similar looking products to the product being viewed. This recommendation type is most useful where images and visual aspects of the products are important parts of the shopping experience.
+
+## How it works
+
+The _Visual similarity_ recommendation type offers recommendations for other products in your catalog that have a visual similarity to the product image currently being viewed. You do not need to do anything to the images in your catalog for this recommendation type to work. Adobe Sensei uses AI to process and analyze the default product images in your catalog and builds attributes used to determine visual similarities. Because this recommendation type is not applicable to most catalogs, it is not enabled by default. You must explicitly enable this recommendation type.
```

Comment:
```suggestion
The _Visual similarity_ recommendation type offers recommendations for other products in your catalog that have a visual similarity to the imagery of the product currently being viewed. You do not need to do anything to the images in your catalog for this recommendation type to work. Adobe Sensei uses AI to process and analyze the default product images in your catalog and builds attributes used to determine visual similarities. Because this recommendation type is not applicable to most catalogs, it is not enabled by default. You must explicitly enable this recommendation type.
```

Reply:
remove "default" for product images